data/entity/npc/monster/undead/corpse/corpse.npcs.toml

```
@@ -0,0 +1,133 @@
+[corpse_spider]
+id = 7914
+hitpoints = 150
+wander_radius = 8
+attack_radius = 4
+max_hit_melee = 15
+style = "slash"
+respawn_delay = 60
+drop_table = "corpse_spider"
+combat_anims = "corpse_spider"
+categories = ["corpse_spider"]
+examine = "Could do with gaining a few pounds."
+
+[crawling_corpse_torso]
+id = 7916
+hitpoints = 350
+wander_radius = 8
+attack_radius = 4
+max_hit_melee = 20
+style = "slash"
+respawn_delay = 60
+drop_table = "crawling_corpse_torso"
+combat_anims = "crawling_corpse_torso"
+categories = ["crawling_corpse_torso"]
+examine = "Could do with gaining a few pounds."
+
+[crawling_corpse_torso_1]
+id = 7917
+clone = "crawling_corpse_torso"
+
+[crawling_corpse_torso_2]
+id = 7918
+clone = "crawling_corpse_torso"
+
+[crawling_corpse_torso_3]
+id = 7919
+clone = "crawling_corpse_torso"
+
+[corpse_archer]
+id = 7923
+hitpoints = 650
+wander_radius = 8
+attack_radius = 4
+max_hit_melee = 19
```

```suggestion
max_hit_range = 19
weapon_style = "bow"
ammo = "steel_arrow"
```

Or whatever the arrow type is supposed to be